packages/fullstack/src/server/mod.rs

```diff
@@ -374,16 +374,7 @@ impl RenderHandleState {
 pub async fn render_handler(
     State(state): State<RenderHandleState>,
     request: Request<Body>,
-) -> impl IntoResponse {
-    // Only respond to requests for HTML
-    if let Some(mime) = request.headers().get("Accept") {
-        let mime = mime.to_str().map(|mime| mime.to_ascii_lowercase());
-        match mime {
-            Ok(accepts) if accepts.contains("text/html") => {}
-            _ => return Err(StatusCode::NOT_ACCEPTABLE),
-        }
-    }
-
+) -> Result<http::Response<axum::body::Body>, StatusCode> {
```

Comment:
Could we keep this as `impl IntoResponse`? This is a public API and I would like to be able to change the error type without a breaking change

Reply:
I moved the type information to `Ok` type we return which seems to fix the issue